networkx/algorithms/flow/gomory_hu.py

```diff
@@ -83,7 +83,7 @@ def gomory_hu_tree(G, capacity="capacity", flow_func=None):
     10
     >>> nx.minimum_cut_value(G, u, v)
     10
-    >>> # The Comory-Hu tree also has the property that removing the
+    >>> # The Gomory-Hu tree also has the property that removing the
```

Comment:
Google:

https://www.google.com/search?q=%22Comory-Hu+tree%22&oq=%22Comory-Hu+tree%22&aqs=chrome..69i57l2j69i59l2.480j0j7&sourceid=chrome&ie=UTF-8

> Showing results for ["Gomory-Hu tree"](https://www.google.com/search?sxsrf=APwXEdcgvIQAbDd5LpjAjt3CTpVFY2XrrA:1687117784876&q=%22Gomory-Hu+tree%22&spell=1&sa=X&ved=2ahUKEwi01ovCy83_AhWMjIkEHam5AoYQkeECKAB6BAgJEAE)
Search instead for ["Comory-Hu tree"](https://www.google.com/search?sxsrf=APwXEdcgvIQAbDd5LpjAjt3CTpVFY2XrrA:1687117784876&q=%22Comory-Hu+tree%22&nfpr=1&sa=X&ved=2ahUKEwi01ovCy83_AhWMjIkEHam5AoYQvgUoAXoECAkQAg)

https://en.wikipedia.org/wiki/Gomory%E2%80%93Hu_tree